packages/lit-html/src/is-server.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const NODE_MODE = false;
+
+export const isServer = NODE_MODE;
```

Comment:
Add JS doc comment to the export and add a @file comment for the file.

Reply:
Done!